### PR TITLE
Feature/execute script asyncronously

### LINKED ISSUE #118: Getting empty results even though there are violations on the page
Sample code

```
import pytest
from selenium import webdriver
from axe_selenium_python import Axe


driver = webdriver.Chrome()
driver.get("http://www.google.com")

axe = Axe(driver)
# Inject axe-core javascript into page.
axe.inject()
# Run axe accessibility checks.
results = axe.execute()
# Write results to file
axe.write_results('a11y.json', results)
driver.close()
# Assert no violations are found
assert len(results["violations"]) == 0, axe.report(results["violations"])
```
This outputs results as {}

But if i axe.run() in the console i get this output:

image

### PR DESCRIPTION
**Pull request by matt-l-w**
Fixes #118 - empty results on chrome driver.

The issue was not using the correct method on the [JavascriptExecutor ](https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/JavascriptExecutor.html#executeAsyncScript-java.lang.String-java.lang.Object...-)interface.  We can run the script asyncronously and return the results using the callback.

I've also added a test case to use a chromedriver.

### CODE REVIEW

**Comment by matt-l-w** [comment]
@moz-hwine / other admin - got a chance to get this working locally but just struggling to get the chromedriver set up on travis.  If anyone gets the chance to have a look that would be fantastic.

**Comment by matt-l-w** [comment]
@moz-hwine - can an admin get to this any time soon?

**Comment by kimberlythegeek** [comment]
@matt-l-w Hi! First I want to say that I really appreciate your contribution! 😄 

Also, my apologies for the delay, I was on leave last week.

I will look over this today or tomorrow. Currently trying to get caught up on emails, and everything else 🙂

**Comment by matt-l-w** [comment]
No worries! Hope the break was good :smiley:

Just checking it wasn't lost in PR purgatory :stuck_out_tongue:

File: README.rst [hunk 1]
```diff
@@ -73,7 +73,11 @@ Contributing
 ------------
 
 Fork the repository and submit PRs with bug fixes and enhancements;
-contributions are very welcome. You can run the tests using
+contributions are very welcome.
+
+Node dependencies must be installed by running `npm install` inside the axe-selenium-python directory.
+
+You can run the tests using
```
Comment (kimberlythegeek) [inline README.rst]: Thanks for this! Sometimes I struggle with writing thorough documentation.

File: axe_selenium_python/axe.py [hunk 2]
```diff
@@ -33,7 +33,8 @@ def execute(self, context=None, options=None):
         :param context: which page part(s) to analyze and/or what to exclude.
         :param options: dictionary of aXe options.
         """
-        template = "return axe.run(%s).then(function(result){return result;});"
+        template = "var callback = arguments[arguments.length - 1];" \
+                   "axe.run(%s).then(results => callback(results))"
```
Comment (kimberlythegeek) [inline axe_selenium_python/axe.py]: Awesome work :)

I don't have a lot of experience with JavaScript; I tried a lot of different approaches and didn't get it to work!

**Pull request merged**